plaidml/bridge/openvino/src/plaidml_plugin/ops/normalize_l2.cpp

```diff
@@ -20,13 +20,15 @@ static OpRegistration reg("normalizel2", [](const Context& ctx) {
   auto I = ctx.operands.at(0);
   std::vector<size_t> axes = get_axis_vector_from_constant_operand(1, ctx.layer);
   auto* layer = dynamic_cast<ngraph::opset1::NormalizeL2*>(ctx.layer);
-  // check if axes is empty
+  auto eps = layer->get_eps();
+  auto eps_mode = layer->get_eps_mode();
+
   if (axes.empty()) {
-    return edsl::make_tuple(I / I);
+    auto norm = I + eps;
+    return edsl::make_tuple(I / norm);
```

Comment:
It's unclear to me if this is the desired behavior; it may be intended to be a broadcast 1 instead. Once the testing infra supports this we should re-examine it. For now it's fine.

Reply:
I agree.